spec/index.bs

```diff
@@ -1264,6 +1264,9 @@ interface GPUAdapter {
     [SameObject] readonly attribute GPUAdapterFeatures features;
     //readonly attribute GPULimits limits; Don't expose higher limits for now.
 
+    readonly attribute unsigned long computeQueueCount;
```

Comment:
So the application doesn't opt-into having more queues, but just receives them, always? That a portability hazard and imho it should be something you request as part of device creation.

Reply:
Even better I think applications should just be able to request a compute or transfer queue, even if the system doesn't have one. The number of queues in the system would just be a hint (or would disappear to avoid fingerprinting).